frontend/src/scenes/organization/Settings/InviteModal.tsx

```diff
@@ -142,17 +149,13 @@ export function BulkInviteModal({ visible, onClose }: { visible: boolean; onClos
                 </div>
             )}
             {!preflight?.email_service_available && (
-                <Alert
-                    type="warning"
-                    style={{ marginTop: 16 }}
-                    message={
-                        <>
-                            Sending emails is not enabled in your PostHog instance.
-                            <br />
-                            Remember to <b>share the invite link</b> with each team member you want to invite.
-                        </>
-                    }
-                />
+                <InfoMessage style={{ marginTop: 16 }}>
+                    <div>
+                        Sending emails is not enabled in your PostHog instance. Remember to{' '}
```

Comment:
Can this message be moved into a component? The same text in two places is bound to lead to someone only changing one of them.

Reply:
Unified.